modules/sdk-coin-tia/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@bitgo/sdk-coin-tia",
+  "version": "1.0.0",
```

Comment:
Is this ready for release upon merge? If not, let's zero out the version number or change access value.